File: Latihan 3/Python/luasS.py
import math

alas = int(input("Masukan Alas     = "))
tinggi = int(input("Masukan tinggi = "))


sisimiring =int(math.sqrt(alas * alas + tinggi * tinggi))
keliling = int(alas + tinggi + sisimiring)

print(f"Sisi miring: {sisimiring}")
print(f"Keliling: {keliling}")

File: Latihan 3/C++/luasS.cpp
#include <iostream>
#include <cmath>


using namespace std;

int main() {
    int tinggi, alas, sisimiring, keliling;

    cout << "Masukan Tinggi Segitiga    : ";
    cin >> tinggi;
    cout << "Masukan Alas Segitiga      : ";
    cin >> alas;

    sisimiring = sqrt(alas * alas + tinggi * tinggi);
    keliling = (alas + tinggi + sisimiring);

    cout << "Sisimiring = " << sisimiring << endl;
    cout << "Keliling = " << keliling ;

    return 0;
}
